rad_gen.py

```diff
@@ -55,63 +58,112 @@
 ##### 3D IC IMPORTS ##### 
 import src.ic_3d.ic_3d as ic_3d
 
+import src.common.constants as consts
+
 
 rad_gen_log_fd = "rad_gen.log"
 log_verbosity = 2
 cur_env = os.environ.copy()
 
+# ██╗      ██████╗  ██████╗  ██████╗ ██╗███╗   ██╗ ██████╗ 
+# ██║     ██╔═══██╗██╔════╝ ██╔════╝ ██║████╗  ██║██╔════╝ 
+# ██║     ██║   ██║██║  ███╗██║  ███╗██║██╔██╗ ██║██║  ███╗
+# ██║     ██║   ██║██║   ██║██║   ██║██║██║╚██╗██║██║   ██║
+# ███████╗╚██████╔╝╚██████╔╝╚██████╔╝██║██║ ╚████║╚██████╔╝
+# ╚══════╝ ╚═════╝  ╚═════╝  ╚═════╝ ╚═╝╚═╝  ╚═══╝ ╚═════╝ 
+
+
+def init_logger():
```

Comment:
I personally come from the philosophy of putting the bare minimum into the "main file" of a program. Basically turning it into:
```
main:
  - parse args
  - res <- run_program
  - parse res messages
  - gracefully return
```
Perhaps this logging code is better suited for utils?